/**:
  ros__parameters:
    voxel_size_x: 0.3
    voxel_size_y: 0.3
    voxel_size_z: 0.1
    x_min: 80.0  # meters
    x_max: 200.0 # meters
    y_min: -20.0 # meters
    y_max: 20.0  # meters